app/move/index.js

```diff
@@ -63,7 +63,11 @@ router.use(
   wizard(createSteps, createFields, createConfig)
 )
 router.get('/:moveId', protectRoute('move:view'), view)
-router.get('/:moveId/confirmation', protectRoute('move:create'), confirmation)
+router.get(
+  '/:moveId/confirmation',
+  protectRoute(['move:create', 'move:review']),
```

Comment:
Why do we need a separate permission for `move:review`? Surely it's akin to create, in practice?

Reply:
Nope, they are different. This permission is for PMU to be allowed to accept or reject a proposed move but we are re-using the confirmation page in that journey. But the permission to create shouldn't allow a user to review a proposed move, likewise, being able to create a move doesn't mean you should be able to review a proposed move.

tl;dr we don't want to give PMU the ability to create single moves, just allocation moves

Reply:
👍